src/app/personside/MainLayout.tsx

```diff
@@ -37,18 +37,61 @@ const DialogPanel = styled.section`
         }
     `;
 
-function MainLayout() {
-    return (
-        <Wrapper>
-            <PersonOversiktsPanel>
-                <VisittkortContainer/>
-                <Lameller/>
-            </PersonOversiktsPanel>
-            <DialogPanel>
-                <ComponentPlaceholder height={'100%'} name={'Dialog Panel'}/>
-            </DialogPanel>
-        </Wrapper>
-    );
+const DialogPanelFixer = styled<{scrollTop: number}, 'div'>('div')`
+          height: 500px;
+          @media(${props => props.theme.media.wideScreen}){
+            display: flex;
+            flex-flow: column nowrap;
+            height: 100%;
+            transform: translateY(${props => props.scrollTop}px);
+          }
+        `;
+
+interface MainLayoutState {
+    scrollTop: number;
+}
+
+interface MainLayoutProps {
+}
+
+class MainLayout extends React.Component<MainLayoutProps, MainLayoutState> {
+    private wrapperRef: HTMLDivElement;
+
+    constructor(props: MainLayoutProps) {
+        super(props);
+        this.state = { scrollTop: 0 };
+    }
+
+    componentDidMount() {
+        this.wrapperRef.addEventListener('scroll', () => this.handleScroll());
+    }
+
+    componentWillUnmount() {
+        this.wrapperRef.removeEventListener('scroll', () => this.handleScroll());
+    }
+
+    handleScroll() { // TODO bør ha en debouncer hvis denne skal brukes
```

Comment:
Hørte dette ble litt diskutert igår, og jeg er i utgangspunktet enig med Eigil at vi bør ungå javascript for å gjøre styling.
Så om vi klarer å løse det på en annen måte så bør vi det.

Har du testa den på nav skrivebord i IE?

Reply:
testet i IE ja. Jepp, naturligvis enig i at styling så langt det lar seg gjøre skal gjøres med css, men denne løsningen krever javascript. Hvis vi finner en bedre løsning uten å bruke javascript er det selvfølgelig bedre :-) Forresten så er dette bare ment som en POC.

Men når det er sagt: forholdet mellom javascript og styling har blitt ganske mye mindre komplekst ved bruk av styled-components og nå blir stylingen i praksis satt i javascipt-filer. Så dersom noe faktisk må løses med js bør det kunne gjøres på en mer elegant måte enn før når vi måtte mappe klassenavn, bruke innline-styling osv.

Reply:
Vi snakket jo om det, men kan legge inn her også at jeg er enig med meg selv. Synes vi bør vurdere å la være å gjøre det hvis det må løses med event listeners.